cmd/docker-app/fork.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"github.com/docker/app/internal/packager"
+	"github.com/docker/cli/cli"
+	"github.com/spf13/cobra"
+)
+
+var (
+	forkMaintainers []string
+	outputDir       string
+)
+
+func forkCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "fork <origin-name> <fork-name> [-p outputdir] [-m name:email ...]",
```

Comment:
Following the flow of `git clone`, can we not use the same name for the fork by default?